Source/WebKit/Scripts/process-entitlements.sh

```diff
@@ -630,8 +632,8 @@ then
     if [[ "${PRODUCT_NAME}" == com.apple.WebKit.WebContent.Development ]]; then true
     elif [[ "${PRODUCT_NAME}" == com.apple.WebKit.WebContent ]]; then ios_family_process_webcontent_entitlements
     elif [[ "${PRODUCT_NAME}" == WebContentExtension ]]; then ios_family_process_webcontent_entitlements
-    elif [[ "${PRODUCT_NAME}" == com.apple.WebKit.WebContent.CaptivePortal ]]; then ios_family_process_webcontent_captiveportal_entitlements
-    elif [[ "${PRODUCT_NAME}" == WebContentCaptivePortalExtension ]]; then ios_family_process_webcontent_captiveportal_entitlements
+    elif [[ "${PRODUCT_NAME}" == com.apple.WebKit.WebContent.CaptivePortal ]]; then ios_family_process_webcontent_lockdown_mode_entitlements
+    elif [[ "${PRODUCT_NAME}" == WebContentCaptivePortalExtension ]]; then ios_family_process_webcontent_lockdown_mode_entitlements
```

Comment:
We should file a bug to change `CaptivePortal` to `LockdownMode` for the process name. We can't do that in a point release since it needs cross-functional changes, but it would be a nice small project for next fall's release.

Reply:
That's a good point, I created rdar://133760617.

Thanks for reviewing!